### ISSUE TITLE
Post service works differently per shift

### ISSUE THREAD

**Issue by aubergine10** [dymanoid/RealTime#245]
So, it's nearly midnight (in game) and I just spotted a mail delivery van (by its headlights) driving down the most dangerous road in my city. This is the view form the driver seat:

![image](https://user-images.githubusercontent.com/1386719/69488279-272a7f80-0e5f-11ea-84c1-4cc3e7e8ee66.png)

This is the road it's on.

![image](https://user-images.githubusercontent.com/1386719/69488298-71136580-0e5f-11ea-8c8b-ded3dea69567.png)

At over 300m above sea level, it winds along a steep mountain face. There's no crash barrier, just a long drop for any vehicle that gets too close to the edge. Literally nobody should be driving down that road at midnight, least of all a van delivering junk mail to a bloke in a fire watch tower.

So, after #243, #242 and #228 discussed recently, I'd like to add Postal Service to the list.

I have no idea how postal service works under the hood, but assuming it delivers to individual buildings (rather than segments of road or something), it could maybe work along these lines...

**Mail delivery vans:***

* Residential: Morning
* Commerical: Day
* Industrial: Day
* Office: Morning, Day
* Other: Evening

They sleep at night. Also, day off on Sundays.

**Mail trucks:**

* Evening, Night, Morning

If possible, deliver to the post offices in morning, collect from post offices in evening. The large sorting office would essentially be churning through mail 24/7, but the smaller post offices would usually close at night.

**Comment by dymanoid** [comment]
Hm... For example, in Germany, the post trucks deliver the mail at night too - not to the end points (post offices), but between the sorting facilities.

I didn't check how the postal service is implemented in the game. No idea whether there is such a thing as "transfer between large sorting centers" at all.

Post delivery at night doesn't make any sense of course when we talk about end customers (city buildings).

**Comment by aubergine10** [comment]
> Hm... For example, in Germany, the post trucks deliver the mail at night too - not to the end points (post offices), but between the sorting facilities.

Yup, that's what I was thinking with having mail **trucks** (which seem to be purely facility-to-facility transit) only work in evenings and night, when the roads are quiet. Big bulk transfers for next-day local delivery by mail **vans** (which seem to be purely local-to/from-customer transit).

There definitely seems to be some distinction between sorted vs. unsorted mail. I have no idea how the sorting facility works, especially considering that even local post offices have some sorting ability.
